test/extension/commandExecutor.test.ts

```diff
@@ -125,5 +128,79 @@ suite("commandExecutor", function() {
                     assert.fail(null, null, "react-native command was not expected to fail");
                 });
         });
+
+        test("getReactNativeVersion should return version string if there is 'react-native' field in project package.json", (done: MochaDone) => {
+            let commandExecutor: CommandExecutor = new CommandExecutor(sampleReactNative022ProjectDir);
+
+            commandExecutor.getReactNativeVersion()
+            .then(version => {
+                assert.equal(version, "^0.22.2");
+            }).done(() => done(), done);
+        });
+
+        test("getReactNativeVersion should return version string if there is 'version' field in react-native package's package.json file", (done: MochaDone) => {
+            const reactNativePackageDir = path.join(sampleReactNative022ProjectDir, "node_modules", "react-native");
+            const commandExecutor: CommandExecutor = new CommandExecutor(sampleReactNative022ProjectDir);
+            let fsHelper: FileSystem = new Node.FileSystem();
+            let versionObj = {
+                "version": "^0.22.0",
+            };
+
+            if (!fs.existsSync(reactNativePackageDir)) {
+                fsHelper.makeDirectoryRecursiveSync(reactNativePackageDir);
+            }
+
+            fs.writeFileSync(path.join(reactNativePackageDir, "package.json"), JSON.stringify(versionObj, null, 2));
+
+            commandExecutor.getReactNativeVersion()
+            .then(version => {
+                assert.equal(version, "^0.22.0");
+                fsHelper.removePathRecursivelySync(path.join(sampleReactNative022ProjectDir, "node_modules"));
```

Comment:
```suggestion
                fsHelper.removePathRecursivelySync(path.join(sampleReactNative022ProjectDir, "node_modules"));
                done();
```

Reply:
It seems that such a behavior courses incorrect test's error handling.